crates/burn-vision/Cargo.toml

```diff
@@ -51,6 +51,7 @@ ndarray = { workspace = true }
 num-traits = { workspace = true }
 paste = { workspace = true }
 serde = { workspace = true }
+rand = { version = "0.9.0", optional = false }
```

Comment:
We already defined rand dependency in the root Cargo file: https://github.com/tracel-ai/burn/blob/main/Cargo.toml#L137

You can use workspace = true

Reply:
Fixed